libraries/Microsoft.Bot.Builder.LUIS/LuisTraceInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+using Microsoft.Cognitive.LUIS;
+using Microsoft.Cognitive.LUIS.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.LUIS
```

Comment:
I don't see any tests around this. Please add a test that: 
1. Populates this class
2. Serializes it to JSON
3. Compares that JSON to reference JSON.

Reply:
Done